ticker-core/ticker-core-quartz/src/main/java/hu/icellmobilsoft/ticker/quartz/service/quartz/SchedulerController.java

```diff
@@ -57,9 +59,9 @@ public class SchedulerController {
      */
     void onStart(@Observes StartupEvent event) {
         List<String> configKeys = timerConfig.activeJobs();
+        jobConfigurationChecker.validateConfiguration(configKeys);
```

Comment:
Ha ezen a ponton elfogadjuk, hogy RTE-t dobunk, akkor a lentebbi try-catch-ben is logozás helyett RTE-be kellene csomagolni a BaseException-t és tovább dobni logozás helyett.